### DISCUSSION TITLE
Training epoch 1 has been running for over 1295 minutes

### DISCUSSION THREAD

**Discussion by juliagorman** [talmolab/sleap#983] [Help!]
Hello, 

I labeled 200 out of 1500 frames on a videowith a single animal and then ran training on it. 1296 minutes later and I am still only on the first epoch. What could be the issue?

Thanks!
Julia

---

# Bug description

## Your personal set up

- OS:

- Version(s):

- SLEAP installation method (listed [here](https://sleap.ai/installation.html#)):
  - [ ] [Conda from package](https://sleap.ai/installation.html#conda-package)
  - [ ] [Conda from source](https://sleap.ai/installation.html#conda-from-source)
  - [ ] [pip package](https://sleap.ai/installation.html#pip-package)
  - [ ] [M1 Macs](https://sleap.ai/installation.html#m1-macs)

<details><summary>Environment packages</summary>

```
# paste output of `pip freeze` or `conda list` here
```

</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```

</details>

**Comment by roomrys** [comment]
Hi @juliagorman,

I edited your response above, do you mind filling out some of the fields (especially any logs from the terminal). Linking a similar issue here: #872.

Thanks,
Liezl

**Reply by roomrys** [reply]
Hi @juliagorman,

Just looping back and checking in. Did you ever manage to overcome this?

Thanks,
Liezl
